src/operation.rs

```diff
@@ -321,7 +321,7 @@ fn process_unary_operation(
                 };
 
                 cases
-                    .remove(&en)
+                    .remove(&en.clone().into())
```

Comment:
Why do we need to clone, convert to then only take a reference back?

Reply:
I was using string before and forgot to roll back this part.